anitya/db/migrations/versions/9262ccab3573_add_check_times.py

```diff
@@ -0,0 +1,41 @@
+"""Add check times
+
+Revision ID: 9262ccab3573
+Revises: 7a8c4aa92678
+Create Date: 2018-10-10 10:11:19.281233
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '9262ccab3573'
+down_revision = '7a8c4aa92678'
+
+
+def upgrade():
+    """ Add next_check and last_check columns to the projects table. """
+    op.add_column(
+        'projects',
+        sa.Column(
+            'last_check',
+            sa.DateTime,
+            server_default=sa.func.current_timestamp()
```

Comment:
According to the PSQL docs this comes with a time zone, as well. If the database server isn't running in UTC as well, you're going to get weird issues with this if you just assume it's UTC.